app/models/concerns/hyrax/solr_document/metadata.rb

```diff
@@ -51,6 +51,7 @@ def self.coerce(input)
         attribute :read_groups, Solr::Array, ::Ability.read_group_field
         attribute :collection_ids, Solr::Array, 'collection_ids_tesim'
         attribute :admin_set, Solr::Array, "admin_set_tesim"
+        attribute :member_ids, Solr::Array, "member_ids_ssim"
```

Comment:
👍

Reply:
I'm assuming that this is to help reduce calls to Fedora.

Reply:
it's a bit weird that this wasn't already exposed as an attribute. it *was* already indexed, so this is just an ergonomics/model-layer-compatibility issue.